src/node/interfaces.cpp

```diff
@@ -155,6 +158,66 @@ class GOVImpl : public GOV
         error = "Governance manager not available";
         return false;
     }
+    GovernanceInfo getGovernanceInfo() override
+    {
+        GovernanceInfo info;
+        if (context().chainman) {
+            const CBlockIndex* tip = WITH_LOCK(::cs_main, return context().chainman->ActiveChain().Tip());
+            int last = 0, next = 0;
```

Comment:
nit: definition of 2 variables in 2 string is not used in codebase; can you split it?